src/config.rs

```diff
@@ -160,6 +161,19 @@ macro_rules! cf_config {
             pub prop_keys_index_distance: u64,
             pub titan: TitanCfConfig,
         }
+
+        impl $name {
+            fn validate(&self) -> Result<(), Box<dyn Error>> {
+                if self.block_size.0 as usize > MAX_BLOCK_SIZE {
+                    return Err(format!(
+                        "invalid block-size {}, exceed max size {}",
```

Comment:
Maybe we should contain the `$name` in the error message to help us to diagnose easier.

Reply:
done